content/docs/command-reference/fetch.md

```diff
@@ -64,10 +64,10 @@ option is used.
 perform data synchronization among local and remote storage. The specific way in
 which the set of files to push/fetch/pull is determined begins with calculating
 file hashes when these are [added](/doc/command-reference/add) with DVC. File
-hash values are stored in the corresponding `dvc.yaml` or `.dvc` files
-(typically versioned with Git). Only the hash specified in `dvc.yaml` or `.dvc`
-files currently in the workspace are considered by `dvc fetch` (unless the `-a`
-or `-T` options are used).
+hash values are stored in the corresponding `dvc.lock` or `.dvc` files
+(typically versioned with Git). Only `dvc.yaml` and `.dvc` files currently in
```

Comment:
in the first line we have `dvc.lock` in the second `dvc.yaml`

Reply:
Applied as many of your recs as possible for fetch+ in a0c869b6, while shortening the Description too (it was too long and somewhat repetitive). Some of them no longer apply (that text was removed).